ui/widgets/mini-profile/src/mini-profile-widget.tsx

```diff
@@ -119,29 +132,53 @@ const ProfileCardWidget: React.FC<unknown> = () => {
   );
 };
 
-// Router is required for the useRouteMatch hook to extract the postId from the url
 const Wrapped = () => {
   const { getTranslationPlugin } = useRootComponentProps();
-  return (
-    <Router>
-      <Routes>
-        {[
-          '@akashaorg/app-akasha-integration/beam/:beamId/*',
-          '@akashaorg/app-akasha-integration/reflect/:reflectionId/*',
-        ].map(r => (
-          <Route
-            key={r}
-            path={r}
-            element={
-              <I18nextProvider i18n={getTranslationPlugin().i18n}>
-                <ProfileCardWidget />
-              </I18nextProvider>
-            }
-          />
-        ))}
-      </Routes>
-    </Router>
+  const rootRoute = createRootRoute({
+    component: Outlet,
+  });
+  //@TODO get base route name from a hook rather instead of hardcoding it
+  const antennaAppBaseRouteName = '@akashaorg/app-akasha-integration';
+  const beamRoutes = (
+    [
+      `${antennaAppBaseRouteName}/beam/$beamId`,
```

Comment:
maybe you can use the routing plugin to access the routes? not sure

Reply:
I think this requires some more thought, currently we don't have a function to get the app name.